structuralcodes/sections/_generic.py

```diff
@@ -208,7 +208,9 @@ def compute_area_moments(geometry, material=None):
             )
             # Change sign due to moment sign convention
             izz *= -1
-            if abs(abs(izy) - abs(iyz)) > 10:
+
+            # Check calculated cross moment
+            if not math.isclose(iyz, izy, rel_tol=1e-9, abs_tol=10):
```

Comment:
ah! you can specify both relative and absolute tolerance then?

I didn't use relative because when they are small numbers it happens that relative tolerance can be high even if for engineering point of view they are basically the same.

This happens for instance whatever units you use (mm or m) when izy and iyz should be near zero for symmetric sections, probably.

I set the absolute to 10 thinking about working in mm (so inertia in mm4). But if you input m, I think 10 is too much? So we should set a value for absolute tolerance proportional to the size of the section.

For instance:
if everything is in mm, your section could be 400 x 600 mm. The inertia would have ordered of magnitude around 1e10 so 10 is about nine orders of magnitude lower. 
if everything is in m, your same section would be 0.4 x 0.6 m so the value 10 is too much: the inertia would be in the order of magnitude of 1e-2 so the absolute value should be around 1e-9? Am I correct?

Reply:
Good point. I have attempted an improvement in 133dbef.